CMakeLists.txt

```diff
@@ -157,6 +162,10 @@ include_directories(
   ${UTEMPTER_INCLUDE_DIR}
   )
 
+IF(NOT ANDROID)
+  include_directories( external/UniversalStacktrace/ust )
```

Comment:
No libexecinfo on Android.

Reply:
ust uses libunwind now, can you try latest master and tell me what error you get if any?

Reply:
Doesn't seem to work, as Android uses the LLVM libunwind but UniversalStacktrace seems to expect the nongnu one? I get errors about missing symbols like `UNW_REG_IP` when compiling, after changing `external/UniversalStacktrace/ust/ust.hpp` to include `unwind.h` instead of `libunwind.h`.